src/libcharon/plugins/kernel_super/oper_db_client/liboper_db_client/liboper_db_client.h

```diff
@@ -14,10 +14,13 @@ typedef struct liboper_db_obj_t liboper_db_obj_t;
 
 enum LIBOPER_DB_CLIENT_CLASS_IDS {
     LIBOPER_DB_CLIENT_CLASS_ID_TERMINATOR,
+    LIBOPER_DB_CLIENT_CLASS_ID_VRF_TUNNEL_COUNT_ID_TERMINATOR
```

Comment:
As I wrote previously, it should be TUNNEL**S** and also probably COUNT_I**N**, so
``` suggestion
    LIBOPER_DB_CLIENT_CLASS_ID_VRF_TUNNELS_COUNT_IN_TERMINATOR
```